admin/class-admin-apple-news.php

```diff
@@ -180,6 +180,19 @@ public function __construct() {
 				apple_news_register_meta_helper( 'post', $post_types, $meta_key, $options );
 			}
 
+			// Prevent Yoast Duplicate Post plugin from cloning apple_news meta.
+			add_filter(
+				'duplicate_post_meta_keys_filter',
+				function( $meta_keys ) {
+					array_filter(
+						$meta_keys,
+						function( $key ) {
+							return substr( $key, 0, 11 ) !== 'apple_news_';
```

Comment:
This solution assumes we'll continue with the convention of prepending `apple_news_` to all the plugin's meta keys.

Another option is to `array_keys( $postmeta )`, assign the resulting array to a class property, and pass that list/array of keys to Yoast in a different hook that accepts a list of postmeta keys to be excluded from the post clone.  More memory usage in that solution but probably fewer cpu cycles on aggregate.  Also potentially safer if somewhere down the line we deviate from the meta naming convention.